pkg/sysinfo/cgroup2_linux.go

```diff
@@ -66,6 +69,25 @@ func newV2(quiet bool, opts *opts) *SysInfo {
 	return sysInfo
 }
 
+func getSwapLimitV2() bool {
+	groups, err := cgroups.ParseCgroupFile("/proc/self/cgroup")
```

Comment:
@AkihiroSuda I'm a bit confused why we need to use libcontainer cgroups here; does containerd cgroups not provide this info? We're already passing that one here;  https://github.com/moby/moby/blob/46cdcd206c56172b95ba5c77b827a722dab426c5/pkg/sysinfo/cgroup2_linux.go#L24

(if it doesn't provide this info, should we fix that there?

Reply:
> The caller never pass "/sys/fs/cgroup/${dockerd_cgroup}". (See `git grep WithCgroup2GroupPath`).
> 
> So we need to detect the cgroup of the daemon by ourselves, regardless to the value of `dirPath`.

Quote from resolved conversations